kubejs/server_scripts/gregtech/Alloys_Recipes.js

```diff
@@ -120,13 +120,17 @@ ServerEvents.recipes(event => {
         .duration(140)
         .EUt(30)
 
-    //Replace default GTCEu glowstone separation recipe to match mixing recipe
-    event.replaceOutput({ id: "gtceu:centrifuge/glowstone_separation" }, 'minecraft:redstone', 'gtceu:tricalcium_phosphate_dust')
-    event.recipes.gtceu.mixer("kubejs:glowstone_dust")
-        .itemInputs('gtceu:tricalcium_phosphate_dust', '#forge:dusts/gold')
-        .itemOutputs('2x minecraft:glowstone_dust')
+    //Replace default GTCEu glowstone mixing/separation recipes
+    event.recipes.gtceu.mixer("gtceu:mixer/glowstone_mixing")
+        .itemInputs('gtceu:tricalcium_phosphate_dust', 'gtceu:barite_dust', '#forge:dusts/gold')
+        .itemOutputs('3x minecraft:glowstone_dust')
```

Comment:
I'm not sure the motivation behind this change, but this locks synthesizing glowstone to post-nether. It also means that barite is no longer something that you can obviously immediately electrolyze in an ore processing setup.

Reply:
In NM, synthesizing Glowstone is a non-issue thanks to the Witch model providing effectively infinite amounts. A visit to the Nether isn't even necessary to get a Witch model running, actually, since you can buy the ore with Monicoins.
In HM/EM, a visit to the Nether is required pre-LV so it's a barrier for all of 30 mins.

Not sure what you're talking about in regards to the ore processing setup, but I do want to point out that electrolyzing Barite results in Barium, Oxygen, and Sulfur. Sulfur and Oxygen can be found in plenty of other places, and Barium is only useful for GT Superconductors (Most players neglect these for EIO/TE supercoductors) and Omnium (You need to make 10-ish stacks & that's it for all modes)

Reply:
> Not sure what you're talking about in regards to the ore processing setup,

There are a bunch of dusts that you get from ore processing where the *only* thing you can do with them is electrolyse (or centrifuge) them (barite is one of them). I'm inclined to immediately process those dusts in my ore processing subnet. By adding an alternate use, one needs to consider whether to keep it around in the unprocessed form.

Reply:
OK, but I don't understand how this would be a bad thing.
Many processing lines start by diverting outputs from oreproc already (Rutile, Bauxite, Uraninite, and more in HM/EM) Heck, Platline requires you to divert Purified Ore, which is in the middle of oreproc instead of the output.

If you're more concerned about not being able to synthesize Barite from Barium/Oxygen/Sulfur despite the singular other source of Barium being Naqline, then that can be solved with a Chemical Reactor recipe.

Reply:
> Many processing lines start by diverting outputs from oreproc already

Sure, but for all of those, the question of "Do I need to divert this?" has the obvious answer of "Yes of course, that is the only (or main) way of getting whatever the product of that processing line". The issue isn't the need to divert something, but rather the difficulty/uncertainty of figuring it out.

It would be one thing if this change improved the gameplay in some fashion, but it seems like it is just to be more "accurate" in describing a fantasy material.